openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -202,3 +205,23 @@ def partial_update(self, request, username):
             )
 
         return Response(account_settings)
+
+
+class AccountDeactivationView(APIView):
+    """
+    Account deactivation viewset. Currently only supports POST requests.
+    Only admins can deactivate accounts.
+    """
+    permission_classes = (permissions.IsAuthenticated, IsStaffUser)
+
+    def post(self, request, username):
+        """
+        POST /api/user/v1/accounts/{username}/deactivate/
+
+        Changes the 'is_active' attribute of a user.
+        """
+        user = User.objects.get(username=username)
+        user.is_active = False
+        user.save()
+        account_settings = get_account_settings(request, [username])[0]
+        return Response(account_settings)
```

Comment:
I'm sending back ``account_settings`` purely arbitrarily (perhaps for a check if the user really has been deactivated?) so I guess I could just change this to return only ``True``?